src/request/request.rs

```diff
@@ -226,7 +241,16 @@ impl<T> Debug for Request<T> {
             .field("headers", self.headers())
             .field("params", &self.head.params)
             .field("cookies", &self.head.cookies)
-            .field("body", self.body())
+            .field("body", &self.body)
             .finish()
     }
 }
+
+impl<T> Pipe for Request<T> {
```

Comment:
This literally exists only for the sake of having a terse echo-server example.

The good news is that `Pipe` no longer requires `Sealed` so users of via will be able to impl `Pipe` for external types.